source/opcode.h

```diff
@@ -130,6 +130,9 @@ bool spvOpcodeIsScalarizable(SpvOp opcode);
 // Returns true if the given opcode is a debug instruction.
 bool spvOpcodeIsDebug(SpvOp opcode);
 
+// Returns true if the given opcode is a commutative instruction.
```

Comment:
I would expand the comment a bit to say that it returns true for exactly those opcodes that are binary operators where the order of the operands is irrelevant.

Reply:
Done.